src/features/movies/MoviePage/index.js

```diff
@@ -84,7 +84,8 @@ export const MoviePage = () => {
                   {cast.map(({ profile_path, name, character, id }) => (
                     <StyledLink to={`/profile/${id}`} key={id}>
```

Comment:
tutaj widze, ze key jest dodane juz, ale w peoplePage w tym miejscu jeszcze nie, nie wiem czy Szymon to bedzie poprawiac, ale jak tylko odpalilam aplikacje to wywalilo na konsole blad ze lista nie ma key w peoplePage

Reply:
Tak, ten key jest potrzebny do wyświetlania podstron. Dlatego jego nie komentowałem i nie chcę usunąć.

Reply:
U siebie w pullu StyledLinki mam juz w common i tam przekazywane jest id do key StyledLinka